### PR TITLE
ADA 26 - attendance record business logic for student

### PR DESCRIPTION
**Pull request by cachemeifyoucan0**
right now, according to [the ticket](https://acmwosu.atlassian.net/browse/ADA-26), it is to be assumed the meeting id exists/is correct

### CODE REVIEW

File: acmw-db-app/pages/api/attendance/[pid].js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    } = req
+
+    let result = {};
+
+    try {
+        if(req.method === 'POST'){
+            const body = typeof(req.body) === 'object' ? req.body : JSON.parse(req.body);
+            switch(pid) {
+                case 'record':
+                    result = await record(body.event_code, body.f_name, body.l_name_dot_num, body.year_level);
```
Comment (SnobbyDragon) [inline acmw-db-app/pages/api/attendance/[pid].js:44]: do we need to check `body` and throw an error before calling `record`

**Review by SnobbyDragon — CHANGES_REQUESTED** [review]

File: acmw-db-app/pages/api/attendance/[pid].js (hunk 1 above)
Comment (cachemeifyoucan0) [reply acmw-db-app/pages/api/attendance/[pid].js:44]: probably a power move I'll add that!

Comment (cachemeifyoucan0) [reply acmw-db-app/pages/api/attendance/[pid].js:48]: gotti

**Review by SnobbyDragon — APPROVED** [review]
awesome sauce

**Pull request merged**